src/components/radios/index.md

```diff
@@ -75,6 +75,10 @@ Remember that on small screens such as mobile devices, the radios will still be
 
 You can add hints to radio items to provide additional information about the options.
 
+Keep each hint to a single, short sentence, without any full stops. Screen readers will read out the entire text when users interact with an item. This could frustrate users if the text is long.
```

Comment:
```suggestion
Keep each hint to a single short sentence, without any full stops. Screen readers will read out the entire text when users interact with an item. This could frustrate users if the text is long.
```